docs/epm/index.asciidoc

```diff
@@ -68,6 +68,17 @@ This indexing strategy has a few advantages:
 * Having a global metrics and logs template, allows to create new indices on demand which still follow the convention. This is common in the case of k8s as an example.
 * Constant keywords allow to narrow down the indices we need to access for querying very efficiently. This is especially relevant in environments which a large number of indices or with indices on slower nodes.
 
+=== Ingest Pipeline
+
+The ingest pipelines for a specific dataset will have the following naming scheme:
+
+```
+{type}-{dataset}-{version}
+```
+
```

Comment:
Could you specify what kind of version this is? I assume it's the package version, but without context it could also be the stack version, or the version of nginx that this package is compatible to, or any other version floating around.

Reply:
It is the package version. I clarified it in my last commit.